src/Atmos/Parameterizations/CloudPhysics/Microphysics.jl

```diff
@@ -1,271 +1,550 @@
 """
     one-moment bulk Microphysics scheme
 
-Microphysics parameterization based on the ideas of Kessler_1995:
+Included processes
   - condensation/evaporation as relaxation to equilibrium
   - autoconversion
   - accretion
-  - rain evaporation
-  - rain terminal velocity
+  - evaporation and sublimation
+  - terminal velocity
 """
 module Microphysics
 
+using SpecialFunctions
+
 using ClimateMachine.MoistThermodynamics
+
 using CLIMAParameters
 using CLIMAParameters.Planet: ρ_cloud_liq, R_v, grav
-using CLIMAParameters.Atmos.Microphysics:
-    C_drag,
-    MP_n_0,
-    K_therm,
-    τ_cond_evap,
-    τ_sub_resub,
-    τ_acnv,
-    q_liq_threshold,
-    E_col,
-    D_vapor,
-    ν_air,
-    N_Sc,
-    a_vent,
-    b_vent
+using CLIMAParameters.Atmos.Microphysics
 
 const APS = AbstractParameterSet
+const ASuspPS = AbstractSuspendedWaterParameterSet
+const AFallPS = AbstractFallingWaterParameterSet
+const ACPS = AbstractCloudParameterSet
+const AIPS = AbstractIceParameterSet
+const ARPS = AbstractRainParameterSet
+const ASPS = AbstractSnowParameterSet
+
+export ζ_rai
+export n0_sno
+export lambda
+export unpack_parameters
+
+export supersaturation
+export G_func
 
-# rain fall speed
-export terminal_velocity_single_drop_coeff
 export terminal_velocity
 
-# rates of conversion between microphysics categories
-export conv_q_vap_to_q_liq
-export conv_q_vap_to_q_ice
-export conv_q_liq_to_q_rai_acnv
-export conv_q_liq_to_q_rai_accr
-export conv_q_rai_to_q_vap
+export conv_q_vap_to_q_liq_ice
+
+export conv_q_liq_to_q_rai
+export conv_q_ice_to_q_sno
+
+export accretion
+export accretion_rain_sink
+export accretion_snow_rain
+
+export evaporation_sublimation
+export snow_melt
 
 """
-    terminal_velocity_single_drop_coeff(ρ)
+    ζ_rai(param_set, ρ)
```

Comment:
Why not leaving the terminal velocity (or fall velocity) name? Seems clearer to me.